### DISCUSSION TITLE
How do I change to colorblind theme?

### DISCUSSION THREAD

**Discussion by ghost** [community/community#15201] [Other Feature Feedback, Questions, & Ideas] [answered]
I disabled colorblind theme. Now I can't find it. I can't enable it.

**Comment by FilipKufalov** [comment]
![image](https://user-images.githubusercontent.com/99423097/164083230-a0407b2c-fff8-43b8-a013-45efe7ed5220.png)
Did you tried clicking on "Browse Additional Color Themes" and typing in "Colorblind" don't include space between 2 words

Edit: I forget to mention you can find themes on extensions

**Comment by queenofcorgis** [comment, marked as answer]
Hi there @eeshvardasikcm - click the dropdown to the right of your profile photo in the upper righthand corner of the page, then click settings (second from the bottom), then on the lefthand bar choose Appearance, you'll have appearance options for both light and dark themes 😄

**Reply by ghost** [reply]
options for both light and dark themes. okay. I will check there. thank you.
